.travis.yml

```diff
@@ -67,38 +43,36 @@ jobs:
     - CC_VER="5"
     - PLATFORM='xenial'
     script: "./travis/build.sh"
-
-  ######################## XENIAL-EMSCRIPTEN
   - stage: build+test
     os: linux
     env:
     - PLATFORM='xenial-emscripten'
     script: "./travis/build.sh"
-
-#  ######################## OSX + TESTS
-#  - stage: build+test
-#    os: osx
-#    if: branch = master
-#    env:
-#    - CMAKE_ARGS='-DBUILD_TESTS=ON'
-#    - TEST='TRUE'
-#    script:
-#    - "./travis/build.sh"
-#
-#  ######################## OSX - PYQRLLIB
-#  - stage: build+test
-#    os: osx
-#    if: branch = master
-#    env:
-#    - CMAKE_ARGS='-DBUILD_PYTHON=ON'
-#    script:
-#    - "./travis/build.sh"
-
-#  ########################
-#  - stage: test_deployment
-#    script: echo "TODO..."
-
-  ########################
+  - stage: build_deb_dist
```

Comment:
This stage is running every time, even for PRs. We dont want to release debs all the time